scripts/download_backtest_data.py

```diff
@@ -8,18 +8,26 @@
 
 from freqtrade import (exchange, arguments, misc)
 
-DEFAULT_DL_PATH = 'freqtrade/tests/testdata'
-
 arguments = arguments.Arguments(sys.argv[1:], 'download utility')
 arguments.testdata_dl_options()
 args = arguments.parse_args()
 
+# Added exchange as a subdirectory to download into
+# as Freqtrad now supports multiple exchanges.
+DEFAULT_DL_PATH = 'freqtrade/tests/testdata/' + args.exchange
```

Comment:
the default download path is not `freqtrade/tests/testdata/` - but `user_data/data` ... and i don't think it was your intend to change this ..

Reply:
That part is pre-existing,  I appended the args.exchange string to the end only. 

This script downloads ticker files, it looks fairly stand-alone